modules/tinymce/CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Indenting or outdenting list items inside a block element that was inside another list item would not work #TINY-7209
 - Switching between unordered and ordered lists would incorrectly alter any parent element that contained that list #TINY-8068
 - The URL detection used for `autolink` and smart paste didn't work if a path segment contained valid characters such as `!` and `:` #TINY-8069
+- Cutting contents to clipboard while selecting between the parent list and a nested list would not always set the list style to `none` on the parent list #TINY-8078
```

Comment:
Not sure about how to describe this to make any sense for anyone reading it in a short sentence.

Reply:
```suggestion
- Cutting content to the clipboard while selecting between the parent list and a nested list would not always set the list style to `none` on the parent list #TINY-8078
```
I can't think of anything better to describe this either, so my suggestion is only for grammar.

Reply:
Yeah, sometimes it's hard to describe these edge case scenarios.